4.0/asvs.py

```diff
@@ -52,10 +54,11 @@ class ASVS:
     asvs_flat = []
     asvs_flat2 = {}
     asvs_flat2['requirements'] = []
+    asvs_opencontrol = {}
 
-    def __init__(self, language):    
-        
-        regex = re.compile('Version (([\d.]+){3})')
+    def __init__(self, language):
+
+        regex = re.compile('Version (([\d.x]+){3})')
```

Comment:
Will this work for any version? 4.x, 4.0, 4.0.2 etc?

Reply:
It should work as it did before except I added the possibility of the version containing an x.
On master, the markdown sets the version as 4.x and this resulted in the code defining no version.
Now it will return 4.x or theoretically 4.0.x if it were defined that way in the markdown.